ocaml/libs/tracing/tracing_export.mli

```diff
@@ -0,0 +1,43 @@
+(*
+* Copyright (C) 2024 Cloud Software Group
+*
+* This program is free software; you can redistribute it and/or modify
+* it under the terms of the GNU Lesser General Public License as published
+* by the Free Software Foundation; version 2.1 only. with the special
+* exception on linking described in file LICENSE.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU Lesser General Public License for more details.
+*)
+
+module Export : sig
+  val set_export_interval : float -> unit
```

Comment:
This lacks documentation

Reply:
https://github.com/xapi-project/xen-api/pull/5551/commits/17fdf8bffb726ba3421a07f8fee35be06a69ad1b Added in here!